### PR TITLE
New Absinthe Sprites

### PR DESCRIPTION
**Pull request by TechnoAlchemisto**
Absinthe
Fixes shotglass absinthe being brown, makes the bottle less eyemelting and more _stylized_.

### CODE REVIEW

File: code/modules/food_and_drinks/drinks/drinks/drinkingglass.dm [hunk 1]
```diff
@@ -613,6 +613,10 @@
 				icon_state = "shotglassred"
 				name = "shot of gibs"
 				desc = "...Let's not talk about this."
+			if ("absinthe")
+				icon_state = "shotglassgreen"
+				name = "shot of absinthe"
+				desc = "I am stuck in the cycles of my guilt..."
```
Comment (Iamgoofball) [inline code/modules/food_and_drinks/drinks/drinks/drinkingglass.dm]: fire up the proc gen boys

**Comment by Cheridan** [comment]
the bottle is still hideous but i guess its an improvement

**Comment by Xhuis** [comment]
Thanks. Bottle looks a lot better than my """"""""""spriting"""""""""" made originally.

**Comment by Chocobro** [comment]
#17975 
>I_have_a_really_tenous_relationship_with_my_parents_after_I_accidentally_killed_my_younger_brother

#17976 
>Why_couldn't_it_have_been_me

hello?

**Pull request merged**